src/drawing/__tests__/DrawingCreator-test.tsx

```diff
@@ -0,0 +1,159 @@
+import React, { MutableRefObject } from 'react';
+import noop from 'lodash/noop';
+import { shallow, ShallowWrapper } from 'enzyme';
+import DrawingCreator, { defaultStroke, Props } from '../DrawingCreator';
+import DrawingSVG from '../DrawingSVG';
+import PointerCapture, { Status } from '../../components/PointerCapture';
+import { Position } from '../../@types';
+import { DrawingPathRef } from '../DrawingPath';
+
+describe('DrawingCreator', () => {
+    const creatorEl = { getBoundingClientRect: () => ({ height: 100, left: 0, top: 0, width: 100 }) };
+    const getDefaults = (): Props => ({
+        onStart: jest.fn(),
+        onStop: jest.fn(),
+    });
+    const getWrapper = (props?: Partial<Props>): ShallowWrapper =>
+        shallow(<DrawingCreator {...getDefaults()} {...props} />);
+    const setDrawingStatus = jest.fn();
+    let capturedPointsRef: MutableRefObject<Array<Position>> = { current: [] };
```

Comment:
It seems like we're duplicating end exposing a lot of the internals of how the component works. Should we mount it, instead?